src/Common/Event_Automator/Power_Automate/Power_Automate_Provider.php

```diff
@@ -150,15 +150,33 @@ protected function add_actions() {
 		// Add endpoints to settings dashboard.
 		add_action( 'admin_init', [ $this, 'add_endpoints_to_dashboard' ] );
 
-		$this->setup_add_to_queues();
+		// Wait until plugins are loaded and then add queues for our various plugins.
+		add_action( 'tribe_plugins_loaded', [ $this, 'setup_add_to_queues' ] );
 	}
 
 	/**
 	 * Adds the actions to add to the queues.
 	 *
 	 * @since 6.0.0 Migrated to Common from Event Automator
+	 * @since TBD Split the method in sub-methods for each plugin.
 	 */
-	protected function setup_add_to_queues() {
+	public function setup_add_to_queues() {
+		$this->add_tec_setup();
+		$this->add_et_setup();
+	}
+
+	/**
+	 * Adds the actions required for The Events Calendar.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
```

Comment:
We don't usually add a `@return` if it's `void` (unless something is changing I'm not aware of)

No need to fix this though.